### PR TITLE
[docs] Spanish translation of pipeline_tutorial.md

### PR DESCRIPTION
**Pull request by aaronjimv**
# What does this PR do?

Add the Spanish version of `pipeline_tutorial.md` to `transformers/docs/source/es`

Note:

There is a link to `pipeline_tutorial#using-pipelines-on-a-dataset` and reading the Spanish version of that doc `es/pipeline_tutorial.md`, I  got the impression that it is incomplete. I would appreciate it if can confirm it.

Thanks.

#28936 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu

### CODE REVIEW

**Comment by aaronjimv** [comment]
cc: @tadeodonegana. @gisturiz

Hello guys, I would appreciate if you could review the translation. Thank you for your help.

**Review by stevhliu — APPROVED** [review]
Thanks for the translation, the docs structure LGTM! We can merge once the content is reviewed :)

If you feel like `pipeline_tutorial.md` is incomplete, please feel free to open a separate PR to complete it!

**Comment by aaronjimv** [comment]
Hello @stevhliu. I am still attentive to this PR, I wrote in the SomosNLP community requesting help to review the content. 

In any case, I will re-tag @tadeodonegana and @gisturiz; I would appreciate your assistance.

Thank you.

**Review by tadeodonegana — APPROVED** [review]
LGTM.

I would only change the word mentioned in the comment below, since `seudocódigo` does not exist in the RAE dictionary. It is just a minimal change, but i think it is correct to change it.

File: docs/source/es/pipeline_webserver.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+#[{"score":0.7742936015129089,"token":1012,"token_str":".","sequence":"test."},...]
+```
+
+¡Y listo, ahora tienes una buena idea de cómo crear un servidor web!
+
+Lo realmente importante es cargar el modelo solo **una vez**, de modo que no haya copias del modelo en el servidor web. De esta manera, no se utiliza RAM innecesariamente. Luego, el mecanismo de queuing (colas) te permite hacer cosas sofisticadas como acumular algunos elementos antes de inferir para usar el agrupamiento dinámico:
+
+<Tip warning={true}>
+
+El ejemplo de código a continuación está escrito intencionalmente como seudocódigo para facilitar la lectura.
```
[the remark this answers is not in the document]
Comment (aaronjimv) [reply docs/source/es/pipeline_webserver.md:82]: Hi @tadeodonegana. Thanks for the feedback! I appreciate your help.

**Review by stevhliu — APPROVED** [review]
Great job and thanks for following up! 🤗

File: docs/source/es/pipeline_webserver.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tadeodonegana) [reply docs/source/es/pipeline_webserver.md]: Thanks to you @aaronjimv , and sorry for the delay on reviewing this!

**Pull request merged**